packages/common/src/authorization/organizationMemberAbility.ts

```diff
@@ -44,8 +39,25 @@ export const organizationMemberAbilities: Record<
             organizationUuid: member.organizationUuid,
         });
     },
-    editor(member, { can }) {
+    interactive_viewer(member, { can }) {
         organizationMemberAbilities.viewer(member, { can });
+        can('create', 'Project', {
+            organizationUuid: member.organizationUuid,
+        });
+        can('create', 'Job');
```

Comment:
Why are Jobs and UnderlyingData actions defined at the org level but not the project level?

Reply:
`Jobs` are not attached to projectUuids, you can create a Job even before there is a project. so it is only an org thing (you can still manage jobs on a project as editor) 

`UnderlyingData` are  defined on projects too https://github.com/lightdash/lightdash/pull/4901/files#diff-756a15380b9eda3bdc4cd927fa88e42c8dc083292e714fc1e30ce4cb9c40b016R33

Reply:
Got it thanks!